app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -927,7 +963,7 @@ public function getPendingLeads(
         $countWithMaxMin = false
     ) {
         $variantIds = ($includeVariants) ? $email->getRelatedEntityIds() : null;
-        $total      = $this->getRepository()->getEmailPendingLeads(
+        $total      = (int) $this->getRepository()->getEmailPendingLeads(
```

Comment:
This doesn't return an integer but an array when $countOnly is false. https://github.com/mautic/mautic/blob/staging/app/bundles/EmailBundle/Entity/EmailRepository.php#L367